libs/island-ui/core/src/lib/DatePicker/DatePicker.tsx

```diff
@@ -17,32 +16,64 @@ import { Text } from '../Text/Text'
 
 import * as styles from './DatePicker.treat'
 import * as coreStyles from './react-datepicker.treat'
+import { Input, InputProps } from '../Input/Input'
+import { VisuallyHidden } from 'reakit'
+
+const languageConfig = {
```

Comment:
Only found one occurence in the project [here](https://github.com/island-is/island.is/blob/ee476d0da2000cb4f854b577a8ca1fea4c6c553c/libs/service-portal/core/src/components/ActionCard/ActionCard.tsx#L45) but I'm wondering if this config should be exported and be reused when possible. It could be moved into the island-ui/utils. What do you think?